main.tf

```diff
@@ -14,7 +14,7 @@ locals {
 }
 
 resource "azurerm_application_gateway" "ag" {
-  name                = "${local.resource_prefix}aks-fe-${format("%02d", count.index)}-${var.env}-agw"
+  name                = var.enable_waf ? "${local.resource_prefix}fe-${format("%02d", count.index)}-${var.env}-agw-waf" : "${local.resource_prefix}aks-fe-${format("%02d", count.index)}-${var.env}-agw"
```

Comment:
is this the way we want to do it? what happens if I want another appgw for something else?